althea_kernel_interface/src/counter.rs

```diff
@@ -69,7 +68,7 @@ impl KernelInterface {
                 "inet6",
                 "counters",
             ],
-        );
+        ).expect("ipset failed to create counter");
```

Comment:
Wouldn't a `?` fit in this and similar calls better? IMO a library crate like this one (and especially as thin as this one) should leave error handling to the caller at all times.

Reply:
actually, I think this error should not be .expected because I think it raises an error if there is already a counter by that name, which will make it crash when it restarts